financial-connections-example/src/main/java/com/stripe/android/financialconnections/example/FinancialConnectionsLauncherActivity.kt

```diff
@@ -38,6 +44,12 @@ class FinancialConnectionsLauncherActivity : AppCompatActivity() {
         }
     }
 
+    override fun onResume() {
+        super.onResume()
+        // prevent playground configuration from leaking to example apps.
+        connectionsDebugSharedPrefs.edit { clear() }
+    }
```

Comment:
Small change on the Playground in the example app to make it work across process kills -> we override a deeplink based on a shared preference for local testing. We were clearing them on Activity's onDestroy to prevent it from leaking to the official examples just in case. 

Process kills kill the activity triggering the preference clear. Moved the clear to the launcher activity instead.